tests/phpunit/unit-tests/admin/class-llms-test-admin-notices.php

```diff
@@ -408,6 +408,41 @@ public function test_has_notice() {
 
 	}
 
+	/**
+	 * Test output_notice().
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_output_notice() {
+
+		LLMS_Admin_Notices::init();
+
+		# Create a normal notice.
+		$notice_html = 'Have you heard of the band 999 MB? They haven\'t got a gig yet.';
+		$notice_id   = 'test-output-notice-normal';
+		LLMS_Admin_Notices::add_notice( $notice_id, $notice_html );
+		LLMS_Admin_Notices::save_notices();
+
+		# Test where current user does not have the 'manage_options' capability.
+		$this->assertOutputEmpty( array( 'LLMS_Admin_Notices', 'output_notice' ), array( $notice_id ) );
+
+		# Test where current user does have the 'manage_options' capability.
+		wp_set_current_user( 1 );
```

Comment:
While this currently (and probably always) will work, I prefer to create a new user when testing admin-related content in favor of relying on user existing, being an admin, and etc...

```suggestion
		wp_set_current_user( $this->factory->user->create( array( 'role' => 'administrator' ) ) );
```